### ISSUE TITLE
Autoresize is not working when inputtextarea is disabled

### ISSUE THREAD

**Issue by ramonchiara** [primefaces/primefaces#631]
Sometimes we want to show a disabled already filled inputtextarea, but although its autoresize is true, it stays small.

**Comment by mertsincan** [comment]
I didn't replicate this issue. Can you please attach a sample test.xhtml, TestBean.java and screenshot  for us to replicate? and What browser and PF version are you using?

**Comment by ramonchiara** [comment]
Sure!

test.xhtml

```
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:h="http://xmlns.jcp.org/jsf/html"
      xmlns:p="http://primefaces.org/ui">
    <h:head>
        <title>Test Disabled InputTextarea</title>
    </h:head>
    <h:body>
        <h:form id="form">
            <p:focus/>
            <p:inputTextarea value="#{rootTest.text}" disabled="true"/>
            <p:inputTextarea value="#{rootTest.text}" />
        </h:form>
    </h:body>
</html>
```

TestBean.java

```
import javax.enterprise.context.RequestScoped;
import javax.inject.Named;

@Named("rootTest")
@RequestScoped
public class TestBean {
    public String getText() {
        return String.format("A text%nwith a%nlot of%nlines so%nI can%ntest if%nautoresize is%ngetting done%nright!");
    }
}
```

Result in Chrome Version 44.0.2403.155 (64-bit)
![image](https://cloud.githubusercontent.com/assets/3186158/9301249/6d83ed0c-44a3-11e5-91b5-766dfce6c33a.png)

I'm using PF version 5.2

```
<dependency>
        <groupId>org.primefaces</groupId>
        <artifactId>primefaces</artifactId>
        <version>5.2</version>
</dependency>
```

**Comment by ramonchiara** [comment]
Note that I used `<p:focus>`
If I don't use it, the autoresize just takes place when I put focus manually on enabled inputtextarea
But, it should be very good if I could get the inputtextarea with the right size when rendered

**Comment by mertsincan** [comment]
I replicated with 5.2. We have some fixes for inputtextarea on 5.2.x and 5.3. Please try this case with  5.2.x or 5.3

**Comment by ramonchiara** [comment]
Sorry didn't give feedback earlier.
I tried with 5.3-SNAPSHOT (2015-10-08) and it's working

![image](https://cloud.githubusercontent.com/assets/3186158/10365671/7881e012-6d9b-11e5-9e03-d6b7b055e220.png)
